package.json

```diff
@@ -14,59 +14,59 @@
     },
     "dependencies": {
         "anymatch": "1.3.0",
-        "autoprefixer-core": "5.1.8",
-        "aws-sdk": "^2.2.9",
+        "autoprefixer": "^6.0.0",
+        "aws-sdk": "^2.19.0",
         "chokidar": "1.6.0",
-        "lodash": "4.15.0",
```

Comment:
Why are these being removed?  I don't want the shared Adobe deps removed, just left at the version they are already at.  We need all these.

NOTE: you not only need to see that your grunt tasks run after you change things, but you should also try running the app.  You'll find that removing stuff will break things.